plugins/repository-gcs/src/test/java/io/crate/gcs/GCSBlobContainerRetriesTests.java

```diff
@@ -0,0 +1,745 @@
+            length = effectiveRangeEnd - rangeStart + 1;
+        } else {
+            length = bytes.length - rangeStart;
+        }
+        exchange.getResponseHeaders().add("Content-Type", bytesContentType());
+        exchange.sendResponseHeaders(HttpStatus.SC_OK, length);
+        int minSend = Math.min(0, length - 1);
+        final int bytesToSend = randomIntBetween(minSend, length - 1);
+        if (bytesToSend > 0) {
+            exchange.getResponseBody().write(bytes, rangeStart, bytesToSend);
+        }
+        if (randomBoolean()) {
+            exchange.getResponseBody().flush();
+        }
+    }
+
+    private String httpServerUrl() {
+        assertThat(httpServer).isNotNull();
+        InetSocketAddress address = httpServer.getAddress();
+        return "http://" + InetAddresses.toUriString(address.getAddress()) + ":" + address.getPort();
+    }
+
+    String downloadStorageEndpoint(String blob) {
+        return "/download/storage/v1/b/bucket/o/" + blob;
+    }
+
+    String bytesContentType() {
+        return "application/octet-stream";
+    }
+
+    Class<? extends Exception> unresponsiveExceptionType() {
+        return StorageException.class;
+    }
+
+    BlobContainer createBlobContainer(
+        final @Nullable Integer maxRetries,
+        final @Nullable TimeValue readTimeout,
+        final @Nullable Boolean disableChunkedEncoding,
+        final @Nullable ByteSizeValue bufferSize
+    ) {
+        final Settings.Builder clientSettings = Settings.builder();
+        clientSettings.put(PROJECT_ID_SETTING.getKey(), "project_id");
+        clientSettings.put(CLIENT_ID_SETTING.getKey(), "client_id");
+        clientSettings.put(CLIENT_EMAIL_SETTING.getKey(), "mail@foobar.com");
+        clientSettings.put(PRIVATE_KEY_ID_SETTING.getKey(), "id1234");
+        clientSettings.put(PRIVATE_KEY_SETTING.getKey(), PKCS8_PRIVATE_KEY);
+        clientSettings.put(ENDPOINT_SETTING.getKey(), httpServerUrl());
+        clientSettings.put(TOKEN_URI_SETTING.getKey(), httpServerUrl() + "/token");
+        if (readTimeout != null) {
+            clientSettings.put(READ_TIMEOUT_SETTING.getKey(), readTimeout);
+        }
+
+        final GCSService service = new GCSService() {
+
+            @Override
+            StorageOptions createStorageOptions(
+                final GCSClientSettings clientSettings,
+                final HttpTransportOptions httpTransportOptions
+            ) {
+                StorageOptions options = super.createStorageOptions(clientSettings, httpTransportOptions);
+                RetrySettings.Builder retrySettingsBuilder = RetrySettings.newBuilder()
+                    .setTotalTimeout(options.getRetrySettings().getTotalTimeout())
+                    .setInitialRetryDelay(Duration.ofMillis(10L))
+                    .setRetryDelayMultiplier(1.0d)
+                    .setMaxRetryDelay(Duration.ofSeconds(1L))
+                    .setJittered(false)
```

Comment:
This is deprecated and not needed anymore, retry always jitter. See https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.retrying.RetrySettings.Builder#com_google_api_gax_retrying_RetrySettings_Builder_setJittered_boolean_